### PR TITLE
Synthesize a volume attribute for the client to filter on.

### PR DESCRIPTION
**Pull request by zerebubuth**
Connects to mapzen/vector-datasource#212.

@rmarianski  could you review, please?

### CODE REVIEW

File: TileStache/Goodies/VecTiles/transform.py [hunk 1]
```diff
@@ -179,8 +179,11 @@ def building_height(shape, properties, fid, zoom):
     height = _building_calc_height(
         properties.get('height'), properties.get('building:levels'),
         _building_calc_levels)
+    area = properties.get('area')
     if height is not None:
         properties['height'] = height
+        if area is not None:
+            properties['volume'] = height * area
```
Comment (rmarianski) [inline TileStache/Goodies/VecTiles/transform.py]: The changes themselves look good to me, but do you think we should rename the function to be a little bit more general? Perhaps we should fold the `min_height` transform into this as well? I can't really think of a good name to capture this though, eg `building_heights_and_volumes`? Or maybe something even less descriptive, eg `building_3d`? :P

Comment (zerebubuth) [inline TileStache/Goodies/VecTiles/transform.py]: I've moved it to a separate function - do you think that's a better way of doing it?

**Comment by rmarianski** [comment]
:+1:

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**